### ISSUE TITLE
[FEATURE] Profiles GUI

### ISSUE THREAD

**Issue by Tigermouthbear** [AresClient/ares#13]
**What version is this applicable?**
both

**Description and explanation of new feature**
A cool gui for managing config profiles. This would be a window on the main clickgui

**Additional context**
basic config saving and loading is already implemented with -save and -load. A gui and possibly other profile data will need to be created.

@Makrennel said theyd be working on this, this issue is to keep track of 2.9 milestones

**Comment by Kore-Development** [comment]
If this is just for settings I would also like to have profiles for enabled mods.

**Comment by Tigermouthbear** [comment]
It saves everything, enabled mods, GUI positions, settings, and anything that passes from one session to another. This is basically just gonna be a GUI interface for the profiles command already in the client.

**Comment by ghost** [comment]
Yep, about the only thing it doesnt do is save settings between multimc instances haha
